conf/rule-type-list.json

```diff
@@ -6,12 +6,12 @@
     ],
     "deprecated": {
         "name": "Deprecated",
-        "description": "These rules have been deprecated in accordance with the <a href=\"/docs/user-guide/rule-deprecation\">deprecation policy</a>, and replaced by newer rules:",
+        "description": "These rules have been deprecated in accordance with the <a href=\"/docs/use/rule-deprecation\">deprecation policy</a>, and replaced by newer rules:",
```

Comment:
```suggestion
        "description": "These rules have been deprecated in accordance with the <a href=\"../use/rule-deprecation\">deprecation policy</a>, and replaced by newer rules:",
```

Reply:
i think it was probably intentional to keep these links absolute. since they're not part of the page source files, it could be confusing to use relative links here. 

would rather keep this as is.

Reply:
Okay, we can discuss this separately from this PR.